.github/workflows/build-distribute.yml

```
@@ -0,0 +1,183 @@
+          echo "${{ secrets.GOOGLE_SERVICES_JSON_XUSHRUKHA_PRODUCTION }}" | base64 --decode > ./app/google-services.json || exit 1
+
+      - name: Decode generic google-services.json for other environments
+        if: inputs.environment != 'NIRAMAY_PRODUCTION' && inputs.environment != 'XUSHRUKHA_PRODUCTION'
+        run: |
+          echo "${{ secrets.GOOGLE_SERVICES_JSON_GENERIC }}" | base64 --decode > ./app/google-services.json || exit 1
+
+      - name: Capitalize first letter of variant
+        run: echo "capitalizedVariant=$(echo '${{ inputs.variant }}' | awk '{print toupper(substr($0,1,1)) substr($0,2)}')" >> $GITHUB_ENV
+      - name: Decode Firebase Credentials
+        run: |
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)